layouts/shortcodes/pages/membership/explore-members.html

```diff
@@ -41,6 +41,7 @@ <h3 class="heading-line text-center">
           <span>{{ i18n "members-guest-members-title" }}</span>
         </h3>
         <ul class="list-inline text-center">
+          <li><a href="https://www.apache.org/" target="_blank"><img src="/images/members/jakarta_member_apache.png" width="115" class="img-responsive"></a></li>
```

Comment:
I'd recommend using width 145 here. While we want to keep them small, we want them to be comparable in size and legible

Reply:
OK, cool.
I tried 145 and 130, I think 130 might be better. 145 looks a little big compared to others.
What do you think?

130:
![image](https://user-images.githubusercontent.com/43075439/118151340-84164e80-b3e1-11eb-9707-b106f9c8622e.png)

145:
![image](https://user-images.githubusercontent.com/43075439/118151352-88db0280-b3e1-11eb-90b2-9c51dc66f759.png)